benchmarks/tpch-gen.sh

```diff
@@ -21,7 +21,12 @@
 pushd ..
 . ./dev/build-set-env.sh
 popd
-docker build -t datafusion-tpchgen:$DATAFUSION_VERSION -f tpchgen.dockerfile .
+
+if [[ -z $1 ]]; then
+    docker build  -f tpchgen.dockerfile -t datafusion-tpchgen:$DATAFUSION_VERSION .
+  else
+    docker build  -f tpchgen.dockerfile -t datafusion-tpchgen:$DATAFUSION_VERSION --build-arg scale_factor=$1 .
```

Comment:
Is there a way we can just pass the scale factor as an argument in `docker run` rather than building the image differently?

Reply:
I'll try

Reply:
fixed @andygrove